server/connectors/bitbucket_connector/webhook_secret.py

```
@@ -0,0 +1,118 @@
+
+
+def get_webhook_secret(org_id: str) -> Optional[str]:
+    """Return the org's webhook secret value, or None when never created."""
+    ref = _get_ref(org_id)
+    if not ref:
+        return None
+    try:
+        from utils.secrets.secret_ref_utils import SecretRefManager
+
+        return SecretRefManager().get_secret(ref)
+    except Exception as exc:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)